service/raft/raft_rpc.hh

```diff
@@ -53,8 +53,8 @@ public:
         const std::vector<raft::config_member>& add,
         const std::vector<raft::server_id>& del) override;
 
-    void add_server(raft::server_address) override;
-    void remove_server(raft::server_id id) override;
+    virtual void on_configuration_change(raft::server_address_set add,
```

Comment:
I believe it's a good habit to put each function parameter to an own line, since function signatures change frequently and having each a parameter on an own line simplifies rebases. I can change this specific place if you insist, but splitting parameters is habitual on my side, so you're asking here for more than fixing this place, you're asking me to drop a useful habit.

Reply:
One man's habit another man's ... I personally disagree. If function signatures change frequently it means the code is either in active development or it is trash. OTOH splitting each argument on a separate line takes up vertical space which is much more valuable than horizontal. Anyway this is not something that should be argued  in case bu case basis but specified in a codding style document and followed.

Reply:
Well, thanks for your opinion, but reformatting/rebasing for the sake of a line break is a bit too much for me. You can open raft.hh and you'll see most lines and comments in this file are short, because they were added or formatted or written by me at some point in time. This line is in the same style as the rest of the code in this file.